front/cypress/e2e/survey_builder/create_and_fill_survey.cy.ts

```diff
@@ -608,14 +552,19 @@ describe.skip('Survey builder', () => {
     cy.get('[data-cy="e2e-short-answer"]').click();
     cy.get('#e2e-title-multiloc').type(page4QnTitle, { force: true });
 
+    // Save the survey and reload the page before adding logic (bug with adding logic)
```

Comment:
We created [a separate task](https://www.notion.so/govocal/Issue-with-not-being-able-to-select-target-pages-in-logic-1e59663b7b268022b1eece8fd51c1926?pvs=4) for this and agreed not to let this block us from releasing the e2e tests.